include/usb/usb_device.h

```diff
@@ -174,8 +174,6 @@ struct usb_interface_cfg_data {
  * may only be updated after calls to usb_deconfig
```

Comment:
Typo in commit msg `usb: Rework the usb core internals.`
s/devie/device/

Reply:
Thanks, will correct this.